### DISCUSSION TITLE
Manim Animation PySR

### DISCUSSION THREAD

**Discussion by tomalexTUD** [MilesCranmer/PySR#727] [General]
Hello Miles Cranmer,

absolutely great Python Package, thanks for doing it open source!

Do you have the code for the animation from pysr_animation.mp4, if its written in Manim. It looks beautiful! I want to adapt it to my pysr use-case. 😄 

Thanks a lot!

**Comment by MilesCranmer** [comment]
Yes! The animation is open-source too :) https://github.com/MilesCranmer/pysr_paper - the manim source is in the animation folder.
